pkgs/development/compilers/polyml/default.nix

```diff
@@ -18,6 +18,11 @@ stdenv.mkDerivation rec {
     "--with-gmp"
   ];
 
+  buildPhase = ''
+    make compiler
+    make compiler
```

Comment:
Why twice?

Reply:
It seemed to me that the commit message has made this clear. Should I copy that explanation into a comment?

Reply:
Actually, I was wrong here. The build scripts used by Isabelle use `make compiler` twice, so that's what I tried at first, but now that I've checked, a single `make compiler` resolves the issue too.

I briefly browsed through PolyML's Makefile and I don't think there's anything that might make the result of a double `make compiler` differ from a single run.

Reply:
Yes please. I don't usually check extended commit messages.